boilerplate/package.json

```diff
@@ -10,14 +10,17 @@
     "patch": "patch-package",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:maestro": "maestro test .maestro/FavoritePodcast.yaml",
+    "test:maestro": "maestro test -e MAESTRO_APP_ID=com.helloworld -e IS_DEV=true .maestro/FavoritePodcast.yaml",
+    "test:maestro:ci": "maestro test -e MAESTRO_APP_ID=com.helloworld -e IS_DEV=false .maestro/FavoritePodcast.yaml",
```

Comment:
These should probably be more generic than running the exact file?

Reply:
We'll also want to set up a couple sub-folders I think. Let me test it out.